src/components/imageContainer/ImageContainer.tsx

```diff
@@ -1,15 +1,52 @@
+import { useState } from 'react';
+import arrow from '../../assets/scroll-arrow.svg';
 import ScrollButton from '../scrollButton/ScrollButton';
+import PhotoIndexIndicator from '../photoIndexIndicator/PhotoIndexIndicator';
+import { st, classes } from './ImageContainer.st.css';
 
 interface IImageContainerProps {
     imgUrls: string[];
 }
-const ImageContainer = ({ imgUrls }: IImageContainerProps) => (
-    <div>
-        <div style={{ backgroundImage: `url(${imgUrls[0]})` }}>
-            <ScrollButton className={''} svgPath={'arrow'} />
-            <ScrollButton className={''} svgPath={'arrow'} />
+const ImageContainer = ({ imgUrls }: IImageContainerProps) => {
+    const [currentImgUrlIndex, setCurrentImgUrlIndex] = useState(0);
+
+    const scrollImg = (action: number) => {
+        const nextIndex = currentImgUrlIndex + action;
+        const imgUrlsLength = imgUrls.length;
```

Comment:
The logic for this algorithim works out, but its unintuitive.
Simpler logic would be to get a boolean (lets call it 'back) instead of anumber

and do somthing like this
```
const nextIndex = currentImgUrlIndex + (back ? -1 : 1)
const newIndex = (nextIndex > 0 && nextIndex < imgUrls.length) ? nextIndex : currentImgUrlIndex)
setCurrentImgUrlIndex(newIndex)
```